test-infra/camel-test-infra-microprofile-lra/src/main/resources/org/apache/camel/test/infra/microprofile/lra/services/container.properties

```diff
@@ -14,4 +14,5 @@
 ## See the License for the specific language governing permissions and
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
-microprofile.lra.container=quay.io/jbosstm/lra-coordinator:5.13.1.Final-2.16.6.Final
\ No newline at end of file
+microprofile.lra.container=quay.io/jbosstm/lra-coordinator:5.13.1.Final-2.16.6.Final
+microprofile.lra.container.ppc64le=icr.io/ppc64le-oss/quarkus/lra-coordinator-quarkus-jvm:latest
```

Comment:
Do we really need to use latest, instead of a fixed version? If you push regularly, this means pulling image on each CI build.